docs/networks.md

```diff
@@ -3,14 +3,13 @@
 
 Networks in the HaaS have three fields:
 
-- 'creator' represents what project created it; or 'admin' if it was
-  administrator-created.  If there are still networks created by a project,
-  then that project cannot be deleted.  Only the creator has the ability to
+- 'owner' represents what project owns it; or 'admin' if it was
```

Comment:
Maybe pass in null instead of a string? The current interface dates back to when everything was form data, so that wasn't an option.